packages/bot-utils/src/help.ts

```
@@ -558,8 +633,8 @@ ${codeBlock(generalHelpTexts.commands.reduce((s, cmd, i) => {
 		const spaces = ' '.repeat(maxCmdLength - cmd.name.length);
 		return (
 			s
-			+ `  ${cmd.name}${spaces}${cmd.short}`
-			+ (i < generalHelpTexts.commands.length - 1 ? '\n' : '')
+				+ `  ${cmd.name}${spaces}${cmd.shortDescription}`
+				+ (i < generalHelpTexts.commands.length - 1 ? '\n' : '')
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)